study.md

```diff
@@ -39,7 +39,7 @@ should be symbols.
 | occupation | 'exterminator' |
 
 ```ruby
-# your answer here
+dale = {"Key" => "Value", "---" => "---", "family_name" => "gribble", "given-name" => "dale", "occupation" => "exterminator"}
```

Comment:
`"Key" => "Value", "---" => "---",` is unnecessary and all of the keys being created are strings.  We want symbols as keys.